docs-markdown/src/controllers/quick-pick-menu-controller.ts

```diff
@@ -97,6 +98,10 @@ export function markdownQuickPick() {
             description: "",
             label: "$(diff) Template",
         });
+        items.push({
+            description: "",
+            label: "$(tasklist) Cleanup...",
+        });
```

Comment:
This can be moved out of the template extension check into the core quickpick menu.